### PR TITLE
Updated call to new date mixin methods

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#7079

*Description of changes:*
- updated call to date mixin
- updated local, minimal date mixin
- updated Storybook (you can ignore these diffs)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/mixins/date-mixin.ts [hunk 1]
```diff
@@ -1,51 +1,81 @@
...
-// This mixin contains only the methods needed for the shared components.
-//
+/**
+ * This is a local, minimal mixin containing only the methods needed for the shared
+ * components when they run within this project (ie, in Storybook).
+ *
+ * When imported into an app project, the shared components should use the mixins
+ * local to that project.
+ */
 @Component({})
```
Comment (severinbeauvais) [inline src/mixins/date-mixin.ts:11]: As per this comment, this mixin only contains the methods needed by components in this library.

(These methods are copied directly from business-filings-ui.)

**Review by andrepestana-aot — APPROVED** [review]
LGTM - Maybe adding a test to check the date would be good but that's ok.

**Comment by severinbeauvais** [comment]
> Maybe adding a test to check the date would be good

Agreed. Fixed!

File: tests/unit/StaffComments.spec.ts [hunk 2]
```diff
@@ -80,6 +80,8 @@ describe('Staff Comments', () => {
     // verify comment in div
     expect(wrapper.findAll('#existing-comments > div').length).toBe(1)
     expect(wrapper.find('#existing-comments > div').text()).toContain('A comment')
+    expect(wrapper.find('#existing-comments > div').text()).toContain('Tester')
+    expect(wrapper.find('#existing-comments > div').text()).toContain('Feb 4, 2021 at 2:08 pm Pacific time')
```
Comment (andrepestana-aot) [inline tests/unit/StaffComments.spec.ts:84]: That's it! 👍

**Pull request merged**